samples/interactive-tutorials/product/set_inventory.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START retail_set_inventory]
+import datetime
+import os
+import time
+
+from google.api_core.client_options import ClientOptions
+from google.cloud.retail import FulfillmentInfo, PriceInfo, \
+    ProductServiceClient, SetInventoryRequest
+from google.cloud.retail_v2 import Product
+from google.protobuf.field_mask_pb2 import FieldMask
+
+from setup.setup_cleanup import create_product, delete_product, get_product
+
+project_number = os.getenv('GOOGLE_CLOUD_PROJECT_NUMBER')
+endpoint = "retail.googleapis.com"
+product_id = "inventory_test_product_id"
```

Comment:
Let's use a random id, which will make the test more stable.

Reply:
fixed